### DISCUSSION TITLE
Aide pour package Canva pro gratuit pendant 1 an pour les étudients

### DISCUSSION THREAD

**Discussion by Elopebi** [community/community#32924] [GitHub Education]
Bonjour,

Je souhaiterais avoir de l'aide au sujet d'un article qui faisait bénéficier les étudients de canva pro gratuitement pendant 1an. Il se trouve que je devais recevoir un lien sur ma boite mail à la suite de la création de mon compte github qui devait me rediriger vers canva, ce que je n'ai jamais recu.
De ce fait, à ce stade je ne sais pas vers quoi me diriger pour bénéficier de canva pro gratuit pour les étudients.

Pouvez-vous m'aider ?

**Comment by Laucobl** [comment]
Hello, 
J'ai dû attendre au moins une semaine avant de recevoir un mail pour confirmer mon inscription.. Mais je n'ai jamais reçu de lien pour avoir mon compte canva pro gratuit pendant 1 an donc j suis dans la même recherche que toi :/

Bon courage...

**Comment by basilems** [comment]
@Elopebi @Laucobl  Apparement l'offre Canva Pro liée au Github Education Pack n'est plus d'actualité depuis au moins août dans cette [discussion](https://github.com/orgs/community/discussions/28136#discussioncomment-3319262).

Si la demande à été faite après, je pense que vous aurez de réponses...

**Comment by samus-aran** [comment]
Hi @elopebi, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

But a quick thank you to @basilems for answering @Elopebi and that we no longer provide a Canva benefit as part of the Global Campus Benefits. However for students who are already on the 12 month benefit (meaning they had already signed up and started to redeem the offer), you will continue to have access for the remainder of your 12 month offer period & can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
